kolla-skip-tags

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
```

Comment:
Doesn't need this does it?

Reply:
This was just to enable syntax highlighting in general. It's getting evaluated by bash / gnu functions, so it seemed reasonable to include.